Tests/GeoLib/TestDuplicateGeometry.cpp

```diff
@@ -0,0 +1,150 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ *
+ */
+
+#include <cstdlib>
+#include <ctime>
+#include <random>
+#include <string>
+
+#include "gtest/gtest.h"
+
+#include "GeoLib/DuplicateGeometry.h"
+#include "GeoLib/GEOObjects.h"
+#include "GeoLib/Point.h"
+#include "GeoLib/Polyline.h"
+#include "GeoLib/Surface.h"
+#include "GeoLib/Triangle.h"
+
+TEST(GeoLib, DuplicateGeometry)
+{
+	GeoLib::GEOObjects geo;
+	std::string input_name ("input");
+
+	// generate points
+	srand ( static_cast<unsigned>(time(nullptr)) );
```

Comment:
All of the system calls do live in the std:: namespace. Please explicitly specify it.